lib/node_modules/@stdlib/_tools/changelog/parse-commits/lib/conventional_changelog.js

```diff
@@ -0,0 +1,274 @@
+					breaking.text += '\n';
+				}
+				breaking.text += node.value;
+			} else if ( node.type === 'breaking-change' ) {
+				startCollecting = true;
+			}
+		}
+
+		/**
+		* Callback invoked upon visiting a text node.
+		*
+		* @private
+		* @param {Object} node - AST node
+		*/
+		function visitTextNode( node ) {
+			breaking.text = node.value;
+		}
+	}
+
+	/**
+	* Callback invoked upon visiting the body node.
+	*
+	* @private
+	* @param {Object} body - AST node
+	*/
+	function onBody( body ) {
+		if ( body ) {
+			visit( body, 'text', extractBody );
+		}
+	}
+
+	/**
+	* Callback invoked upon visiting a text node.
+	*
+	* @private
+	* @param {Object} node - AST node
+	*/
+	function extractBody( node ) {
+		if ( out.body !== '' ) {
+			out.body += '\n';
+		}
+		out.body += node.value;
+	}
+
+	/**
+	* Callback invoked upon visiting the summary node.
+	*
+	* @private
+	* @param {Object} summary - AST node
+	*/
+	function onSummary( summary ) {
+		visit( summary, visitSummary );
+	}
+
+	/**
+	* Processes a summary node and extracts relevant information.
+	*
+	* @private
+	* @param {Object} node - AST node
+	*/
+	function visitSummary( node ) {
+		switch ( node.type ) {
+		case 'type':
+			out.type = node.value;
+			out.header += node.value;
+			break;
+		case 'scope':
+			out.scope = node.value;
+			out.header += '(' + node.value + ')';
+			break;
+		case 'breaking-change':
+			out.header += '!';
+			break;
+		case 'text':
+			out.subject = node.value;
+			out.header += ': ' + node.value;
+			break;
+		default:
+			break;
+		}
+	}
+
+	/**
+	* Processes a footer node and extracts relevant information.
+	*
+	* @private
+	* @param {Object} node - AST node
+	*/
+	function processFooter( node ) {
+		var hasRefSeparator = false;
+		var coAuthoredBy = false;
+		var closesIssue = false;
+		var reference = {};
+		var hasPRURL = false;
+
+		visit( node, [ 'type', 'separator', 'text' ], processFooterNode );
+		if ( hasRefSeparator && RE_DIGITS.test( reference.ref ) ) {
+			reference.prefix = '#';
+			out.references.push( reference );
+		}
+		if ( hasPRURL || closesIssue || coAuthoredBy ) {
+			out.mentions.push( reference );
+		}
+
+		/**
+		* Processes a footer node and extracts relevant information.
+		*
+		* @private
+		* @param {Object} node - AST node
+		*/
+		function processFooterNode( node ) {
+			switch ( node.type ) {
+			case 'type':
+				if ( node.value === 'PR-URL' ) {
+					hasPRURL = true;
+				} else if ( RE_CLOSES.test( node.value ) ) {
+					closesIssue = true;
+				} else if ( RE_CO_AUTHORED_BY.test( node.value ) ) {
+					coAuthoredBy = true;
+				}
+				reference.action = node.value;
+				break;
+			case 'separator':
+				if ( node.value.includes( '#' ) ) {
+					hasRefSeparator = true;
+				}
+				break;
+			case 'text':
+				if ( node.value.includes( options.issueURL ) ) {
+					reference.url = node.value;
+					reference.ref = node.value.split( '/' ).pop();
+				} else if ( node.value.includes( options.prURL ) ) {
```

Comment:
```suggestion
				if ( node.value.includes( options.issueURL ) ) {
					reference.url = node.value;
					reference.ref = node.value.split( '/' ).pop();
				} else if ( node.value.includes( options.prURL ) ) {
```

You're using `#includes`. Switch to ES5 equivalent, such as `#indexOf` or a stdlib package equivalent.

Reply:
@kgryte Replaced. Ready for final approval now.